docs/tutorials/load-test-grpc.md

```diff
@@ -51,8 +53,8 @@ Load test a Kubernetes gRPC service and validate its [service-level objectives (
     --set ready.timeout=60s \
     --set grpc.host="routeguide.default:50051" \
     --set grpc.call="routeguide.RouteGuide.ListFeatures" \
-    --set grpc.protoURL="https://raw.githubusercontent.com/grpc/grpc-go/master/examples/route_guide/routeguide/route_guide.proto" \
     --set grpc.dataURL="https://raw.githubusercontent.com/iter8-tools/docs/main/samples/grpc-payload/server.json" \
+    --set grpc.protoURL="https://raw.githubusercontent.com/grpc/grpc-go/master/examples/route_guide/routeguide/route_guide.proto" \
```

Comment:
If the intent is to have the order of inputs the same in the various examples, `protoURL` and `dataURL` should be switched in the *Unary example*